website/blog/2020-11-04-2.2.0.md

```diff
@@ -0,0 +1,820 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.2: espree parser, TS 4.1 and ESM bundles for browsers"
+---
+
+This release supports [espree](https://github.com/eslint/espree) parser, supports TypeScript 4.1 via [Babel 7.12](https://babeljs.io/blog/2020/10/15/7.12.0), ships ESM bundles for browsers, and includes many bug fixes and improvements!
+
+<!--truncate-->
+
+## Highlights
+
+### JavaScript
+
+#### Add `espree` parser ([#9000](https://github.com/prettier/prettier/pull/9000) by [@fisker](https://github.com/fisker))
+
+A new value for the `parser` option has been added: [`espree`](https://github.com/eslint/espree) - which is the [`ESLint`](https://github.com/eslint/eslint) default parser.
+
+Note that `espree` only works for [ECMAScript Finished Proposals](https://github.com/tc39/proposals/blob/master/finished-proposals.md), and is stricter than the `babel` parser.
+
+### TypeScript
+
+#### Support TypeScript 4.1 via Babel ([#9473](https://github.com/prettier/prettier/pull/9473) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+`--parser=babel-ts` is required.
```

Comment:
Maybe worth including a sentence on the future plans? As a Prettier+TS user who is not rushing to try 4.1 features, should I do anything with my projects’ configs?

Reply:
I tried to improve the changelog. WDYT? https://github.com/prettier/prettier/pull/9589/commits/5091e02eafe2c37c1422ccae2d7d6184e4930505
/cc @fisker @thorn0

Reply:
That's clearly better now! One extra thing I would add is a sentence on the expected longevity of this override. As a developer who is planning to use TS 4.1, 4.2 and so on in the future, do I have to keep the override in my config file from now on and forever? Or is it just some temporary thing until, say, Prettier v2.3 is released, after which I can remove the hack?

Personally, I’m not rushing to use TS 4.1 features, but I might need them in a couple of months. The practical question I have is if I should bother updating my shared prettier config with this override or just wait for the next release of Prettier.

Reply:
Thank you! That's certainly important. I've fixed 09f0c22